packages/cli/src/WorkflowRunner.ts

```diff
@@ -54,12 +56,48 @@ export class WorkflowRunner {
 		private readonly workflowStaticDataService: WorkflowStaticDataService,
 		private readonly nodeTypes: NodeTypes,
 		private readonly permissionChecker: PermissionChecker,
+		private readonly ownershipService: OwnershipService,
 	) {
 		if (this.executionsMode === 'queue') {
 			this.jobQueue = Container.get(Queue);
 		}
 	}
 
+	async init() {
+		if (this.executionsMode === 'queue') return;
+
+		const executionIds = await this.executionRepository.getNewExecutionIds();
+		for (const executionId of executionIds) {
+			this.logger.debug('Resuming enqueued execution', { executionId });
+			// Don't block here by using await
+			void this.reEnqueueExecution(executionId);
+		}
+	}
+
+	private async reEnqueueExecution(executionId: string) {
+		const fullExecutionData = await this.executionRepository.findSingleExecution(executionId, {
+			includeData: true,
+			unflattenData: true,
+		});
+
+		if (!fullExecutionData) {
+			throw new ApplicationError('Execution does not exist.', { extra: { executionId } });
+		}
+
+		const workflowId = fullExecutionData.workflowData.id;
+		const user = await this.ownershipService.getWorkflowOwnerCached(workflowId);
```

Comment:
But how do we know it was the workflow owner who had originally commissioned the execution we're re-enqueuing?

Plus, in the near future with RBAC P2 there won't even be a workflow owner user anymore, but rather a workflow owner project. Is there a way we can avoid referring to a user here? (Without a larger refactor.)